# Uart Parity types
# uart_parity_t - enumeration
uint8 value

#Pre-Defined ENUM Constants
	uint8 uart_parity_Current = 0 #  Use the current Parity on selected port
	uint8 uart_parity_Odd     = 1
	uint8 uart_parity_Even    = 2
	uint8 uart_parity_Mark    = 3
	uint8 uart_parity_Space   = 4
	uint8 uart_parity_None    = 5
